enemy-data/abyss-order/abyss-lector-violet-lightning.md

```diff
@@ -50,5 +56,15 @@ These are the evangelists and the scholars of the Abyss. Their violent lightning
 | 95 | 221991 |
 | 100 | 294121 |
 
-## Attacks
+## Attacks  
+* **Thunderbolt:** Summons a falling bolt of Electro energy to strike its foe. When below 50% HP it will summon 2 bolts consecutively. When its shield is active it will summon 3 bolts consecutively. The visual indicator for this attack will appear at the active character's feet.
+
+* **Thunder Wave:** Conjures a wave of lightning bolts striking down in a line towards the player. When it's shield is active, it will unleash two waves of lightning at once, usually from different directions but sometimes in a straight line.
+
+* **Thunderous Declaration:** When a character strays near to the Abyss Lector, it will implode itself with a powerful electromagnetic blast.
+
+* **Storming Sanctum:** After falling to 20% health, the Abyss Lector will remove his remaining health to imbue itself with a powerful Electro shield. Breaking this shield will kill the Lector. Once the shield is activated, nearby players are knocked back from the Lector and damaged. The Lector also gains new/upgraded attacks when it is equipped with the Electro shield.
+
+* **Abyssal Light:** Conjures 3 Electro orbs that launch towards the player's current position. If hit by these orbs, your energy will be depleted by a flat amount unless they're protected by a shield.
```

Comment:
arbitrated pronoun change?
"Your energy" should be "the team's energy" or "the character's energy"